sktime/tests/_config.py

```diff
@@ -44,7 +43,6 @@
     "MLPClassifier",
     "MLPRegressor",
     "CNNRegressor",
-    "ResNetRegressor",
```

Comment:
I think you fixed `SimpleRNNRegressor`, `SimpleRNNRegressor`, but not `ResNetRegressor`, so:

* if you reacivate the tests for the `ResNetRegressor`, it will fail due to the `lambda` issue. So, you may like to leave the `ResNetRegressor` tests deactivated until all `ResNetRegressor` related failures get fixed.
* on the other hand, you may like to reactivate `SimpleRNNClassifier` tests, by removing it from the exception list too.